target/linux/ipq806x/image/generic.mk

```diff
@@ -547,6 +547,29 @@ define Device/ubnt_unifi-ac-hd
 endef
 TARGET_DEVICES += ubnt_unifi-ac-hd
 
+define Device/xiaomi_mi-router-hd
+	$(call Device/LegacyImage)
+	$(Device/dsa-migration)
```

Comment:
New device no migration?

Reply:
My firmware for this router has been used by many users for a long time. My patch was added to Chinese forks a long time ago.
And with kernel version 6.x, DSA support appeared.

Reply:
In that case remove it, forks that supported the device dont matter for this.

Reply:
> In that case remove it, forks that supported the device dont matter for this.

Because of this solution, users may encounter problems when installing official OpenWRT firmware.
Why do this?

Reply:
Row `$(Device/dsa-migration)` removed